maquetado/scss/styles.scss

```diff
@@ -34,6 +35,20 @@
   width: 100%;
 }
 
+.navbar {
+  height: 100%;
+  max-height: $navbar-height;
+  padding: 0 155px;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 999;
```

Comment:
z-indexes are hard to handle at large scale apps, hardcoding the values makes it harder. A good practice is to have a scss file where you store all of your layers to ensure that none of your components break. https://css-tricks.com/handling-z-index/